forescout/assets/dashboards/forescout_threat_protection.json

```
@@ -0,0 +1,2461 @@
+{
+    "title": "Forescout Threat Protection",
+    "description": "This dashboard provides insights into threat protection activities, including suspicious connections, scans, lockdowns, blocks, and email worm detections.",
+    "widgets": [
+        {
+            "id": 2986136853484646,
+            "definition": {
+                "type": "image",
+                "url": "https://www.forescout.com/wp-content/uploads/2019/01/forescout-logo.svg",
+                "url_dark_theme": "https://docs.forescout.com/assets/img/logo-forescout.svg",
+                "sizing": "contain",
+                "margin": "md",
+                "has_background": true,
+                "has_border": true,
+                "vertical_align": "center",
+                "horizontal_align": "center"
+            },
+            "layout": {
+                "x": 0,
+                "y": 0,
+                "width": 5,
+                "height": 3
+            }
+        },
+        {
+            "id": 4005569029054089,
+            "definition": {
+                "type": "note",
+                "content": "This dashboard offers a comprehensive summary of threat protection activities, including suspicious connection attempts, scan events, lockdown actions, block events, and email worm detections.\n\nFor more information, see the [Forescout Integration Documentation](https://docs.datadoghq.com/integrations/forescout/).\n\n**Tips**\n- Use the timeframe selector in the top right of the dashboard to change the default timeframe.\n- Clone this dashboard to rearrange, modify and add widgets and visualizations.",
```

```suggestion
                "content": "This dashboard offers a comprehensive summary of threat protection activities, including suspicious connection attempts, scan events, lockdown actions, block events, and email worm detections.\n\nFor more information, see the [Forescout Integration Documentation](https://docs.datadoghq.com/integrations/forescout/).\n\n**Tips**\n- Use the timeframe selector in the top right of the dashboard to change the default timeframe.\n- Clone this dashboard to rearrange, modify, and add widgets and visualizations.",
```
Same note about link

---

Done and replied for link related question above